wdl-lint/src/rules/import_sort.rs

```diff
@@ -57,11 +65,11 @@ impl Rule for ImportSortRule {
     }
 
     fn tags(&self) -> TagSet {
-        TagSet::new(&[Tag::Style, Tag::Clarity])
+        TagSet::new(&[Tag::Style, Tag::Clarity, Tag::Sorting])
     }
 
     fn exceptable_nodes(&self) -> Option<&'static [SyntaxKind]> {
-        Some(&[SyntaxKind::VersionStatementNode])
+        Some(&[SyntaxKind::ImportStatementNode])
```

Comment:
Please revert this change.

_Every_ rule should have `VersionStatementNode` as an "exceptable node"*, as placing an "except statement" above the version statement is how we allow users to "shut off" a lint for the entire document. In this case, since imports should only appear at the top of a document, that's the only place we should allow users to toggle this rule.

*we should probably add a test to ensure this is true